### PR TITLE
[10.0][IMP] shopinvader_guest_mode: Allow to bind a guest to real

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

File: shopinvader_guest_mode/wizards/partner_binding_line.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        res = super(ShopinvaderPartnerBindingLine, self).action_apply()
+        for record in self.filtered(lambda r: r.bind and r.is_guest):
+            backend = (
+                record.shopinvader_partner_binding_id.shopinvader_backend_id
+            )
+            partner_binding = record.partner_id.shopinvader_bind_ids.filtered(
+                lambda x, b=backend: x.backend_id == b
+            )
+            if partner_binding:
+                partner_binding.write({"is_guest": False})
```
Comment (Cedric-Pigeon) [inline shopinvader_guest_mode/wizards/partner_binding_line.py:29]: you should also reset `expiry_dt`

**Review by Cedric-Pigeon — CHANGES_REQUESTED** [review]

File: shopinvader_guest_mode/wizards/partner_binding_line.py (hunk 1 above)
Comment (Cedric-Pigeon) [reply shopinvader_guest_mode/wizards/partner_binding_line.py:29]: well, sorry missed it is a computed field

**Comment by rousseldenis** [comment]
@Cedric-Pigeon Could you update your review?

**Review by Cedric-Pigeon — APPROVED** [review]
Code review and functional tests

**Review by hparfr — APPROVED** [review]

**Comment by hparfr** [comment]
Pre-commit is failing

**Comment by rousseldenis** [comment]
> Pre-commit is failing

I'll rebase

**Pull request merged**